frontend/src/views/BookingView.vue

```diff
@@ -138,7 +138,7 @@ const handleError = (data) => {
 
   if (data?.detail?.id === 'SCHEDULE_NOT_ACTIVE') {
     errorHeading.value = '';
-    errorBody.value = data.value.detail.message;
+    errorBody.value = data?.detail.message;
```

Comment:
This is just a legitimate error I had to fix while testing.

Reply:
Good catch